### PR TITLE
Flu only bugfix - Inconclusive test results

### LINKED ISSUE #6239: [BUG] Flu-only submitting covid results
Discovered this while doing smoke tests pre adding flu only devices.
When submitting flu only test results an "UNDETERMINED" covid result was also being submitted.

To reproduce: add a flu-only test device, submit the test card with the inconclusive box checked. Verify via dev tools network tab that 3 results were submitted including a covid test.

### PR DESCRIPTION
**Pull request by DanielSass**
# FRONTEND PULL REQUEST

## Related Issue

- resolves #6239 

## Changes Proposed

- Bug: `UNDETERMINED` covid results were being included on inconclusive flu only test card submissions.  Adjusting validation and default behavior to ensure that doesn't happen and adding a unit test to cover the case.

## Testing

- How should reviewers verify this PR?

## Screenshots / Demos

- For large changes, please pair with a designer to ensure changes are as intended

### CODE REVIEW

File: frontend/src/app/testResults/MultiplexResultInputForm.test.tsx [hunk 1]
```diff
@@ -184,6 +184,36 @@ describe("TestResultInputForm", () => {
...
+    expect(screen.getAllByLabelText("Positive (+)")[0]).not.toBeChecked();
+    expect(screen.getAllByLabelText("Positive (+)")[1]).not.toBeChecked();
+    expect(screen.getAllByLabelText("Negative (-)")[0]).not.toBeChecked();
+    expect(screen.getAllByLabelText("Negative (-)")[1]).not.toBeChecked();
+    expect(
+      screen.getByRole("checkbox", { name: /mark test as inconclusive/i })
+    ).toBeChecked();
+    expect(screen.getByText("Submit")).toBeEnabled();
+  });
+
```
Comment (DanielSass) [inline frontend/src/app/testResults/MultiplexResultInputForm.test.tsx:216]: Reverted the fix to verify that this test would previously have failed.

**Review by zdeveloper — APPROVED** [review]
Changes LGTM! deploying it to `dev7`

**Review by nathancrtr — APPROVED** [review]
This appears to still in be draft, but I've tested on `dev7` and all looks good.

**Before**:
Screen Shot 2023-08-02 at 10 26 18 AM

**After**:
Screen Shot 2023-08-02 at 10 26 28 AM

**Review by mpbrown — APPROVED** [review]
LGTM! Thanks for the regression test too!

**Pull request merged**